src/components/story-description/story-description.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC, useMemo } from 'react';
+import cx from 'classnames';
+import styles from './story-description.module.scss';
+import { ElementContent } from '../../types';
+import { FlexAlign, FlexDirection, FlexGap, FlexJustify } from 'src/helpers/components/Flex/FlexConstants';
+import Flex from 'src/helpers/components/Flex/Flex';
+
+type StoryDescriptionProps = {
+  align?: FlexAlign;
```

Comment:
Let's also add this enum to static props, so we'd be able to reference it like this `StoryDescription.align.START`